geom/API/grid_get.F90

```diff
@@ -0,0 +1,45 @@
+#include "MAPL_Generic.h"
+
+submodule (mapl3g_Geom_API) grid_get_smod
+
+   use mapl_ErrorHandling
+   use mapl3g_VectorBasis, only: GridGetCoords
+
+   implicit none
+
+contains
+
+   module subroutine grid_get(grid, unusable, im, jm, latitudes, longitudes, rc)
+      use mapl_KeywordEnforcer
+      type(ESMF_Grid), intent(in) :: grid
+      class(KeywordEnforcer), optional, intent(in) :: unusable
+      integer, optional, intent(out) :: im
+      integer, optional, intent(out) :: jm
+      real(kind=ESMF_KIND_R4), optional, pointer, intent(out) :: latitudes(:,:)
+      real(kind=ESMF_KIND_R4), optional, pointer, intent(out) :: longitudes(:,:)
```

Comment:
I think we have to support getting these in R8 as well.  Users won't usually care, but they are stored in double precision, and it can matter for some things.  Separate PR - I'm just requesting you open an issue.

Reply:
Create #3590